Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -396,14 +386,55 @@ def setCoordinates(self, glyphName, coord, ttFont):
 			# https://github.com/fonttools/fonttools/pull/1198
 			horizontalAdvanceWidth = 0
 		leftSideBearing = otRound(glyph.xMin - leftSideX)
-		ttFont["hmtx"].metrics[glyphName] = horizontalAdvanceWidth, leftSideBearing
+		hMetrics[glyphName] = horizontalAdvanceWidth, leftSideBearing
 
-		if "vmtx" in ttFont:
+		if vMetrics is not None:
 			verticalAdvanceWidth = otRound(topSideY - bottomSideY)
 			if verticalAdvanceWidth < 0:  # unlikely but do the same as horizontal
 				verticalAdvanceWidth = 0
 			topSideBearing = otRound(topSideY - glyph.yMax)
-			ttFont["vmtx"].metrics[glyphName] = verticalAdvanceWidth, topSideBearing
+			vMetrics[glyphName] = verticalAdvanceWidth, topSideBearing
+
+
+	# Deprecated
```

Comment:
Wait, wait, wait, why are these methods deprecated? Because they shouldn't be used publicly?

Reply:
Oh I see you just said something about that.